### PR TITLE
Add drivers for T5 Samsung Portable SSD.

### PR DESCRIPTION
**Pull request by kdeldycke**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask-drivers/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by kdeldycke** [comment]
First draft for this new cask. Thorough checks are in progress...

File: Casks/samsung-portable-ssd-t5.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+cask 'samsung-portable-ssd-t5' do
+  version '1.6.7'
+  sha256 'c53ea42eac82fa6c8040daa35e717712d3ac09912bb777467607fef49fff1b88'
+
+  # Samsung site link directly to AWS S3 asset and do not provide any redirect
+  # from its main domain.
```
Comment (miccal) [inline Casks/samsung-portable-ssd-t5.rb:6]: These first two comments aren't necessary -- the third and final comment is enough.

Comment (kdeldycke) [reply Casks/samsung-portable-ssd-t5.rb]: Thanks @miccal for the feedback! Fixed in: https://github.com/Homebrew/homebrew-cask-drivers/pull/1600/commits/3d7c43f6dd1b548bb5e789cbab76dd4bf4d89481

**Comment by kdeldycke** [comment]
I performed all checks and tests, and the build pass (except for the repository-wide `Calling brew pull is deprecated! Use hub checkout instead.` error).

This PR is redy to be merged! :)

File: Casks/samsung-portable-ssd-t5.rb (hunk 1 above)
Comment (miccal) [reply Casks/samsung-portable-ssd-t5.rb]: No problem -- you will also need to remove the commented lines in `uninstall delete:`.

Comment (kdeldycke) [reply Casks/samsung-portable-ssd-t5.rb]: @miccal done in https://github.com/Homebrew/homebrew-cask-drivers/pull/1600/commits/065ba01c25f1c202cbc78683ba13f77d566f2ecb

**Comment by kdeldycke** [comment]
Thanks @victorpopkov for the merge!

**Pull request merged**